index.js

```diff
@@ -107,15 +107,24 @@ class UpdateNotifier {
 			};
 		});
 	}
+	isYarnGlobal() {
+		const isWindows = process.platform === 'win32' || process.env.OSTYPE === 'cygwin' || process.env.OSTYPE === 'msys';
+		const yarnPath = isWindows ? path.join('Yarn', 'config', 'global') : path.join('.config', 'yarn', 'global');
+		if (process.cwd().includes(yarnPath)) {
+			return true;
+		}
+		return false;
+	}
 	notify(opts) {
 		if (!process.stdout.isTTY || isNpm() || !this.update) {
 			return this;
 		}
 
 		opts = Object.assign({isGlobal: isInstalledGlobally()}, opts);
+		const installCommand = this.isYarnGlobal() ? 'yarn global upgrade ' + this.packageName : 'npm i ' + (opts.isGlobal ? '-g ' : '') + this.packageName;
```

Comment:
`yarn global upgrade <package>` will not upgrade package to latest version. Should be `yarn global add <package>` or `yarn global upgrade <package>@latest`.